thetis/solver.py

```diff
@@ -551,6 +629,11 @@ def create_fields(self):
         self.fields.h_elem_size_3d = Function(self.function_spaces.P1)
         self.fields.h_elem_size_2d = Function(self.function_spaces.P1_2d)
         get_horizontal_elem_size_3d(self.fields.h_elem_size_2d, self.fields.h_elem_size_3d)
+        self.set_sipg_parameter()
+        self.fields.sipg_parameter = self.options.sipg_parameter
+        self.fields.sipg_parameter_vertical = self.options.sipg_parameter_vertical
+        self.fields.sipg_parameter_tracer = self.options.sipg_parameter_tracer
+        self.fields.sipg_parameter_vertical_tracer = self.options.sipg_parameter_vertical_tracer
```

Comment:
You don't need to add these in `self.fields` as SIPG parameter is accessed from `self.options`.